tests/FakeItEasy.Net35.Tests/ZipTests.cs

```diff
@@ -15,12 +19,10 @@ public void Should_zip_correctly()
             var numbers = new[] { 1, 2 };
 
             // Act
-            var zipped = letters.Zip(numbers, (f, s) => new { Letter = f, Number = s }).ToList();
+            var zipped = letters.Zip(numbers, (l, n) => l + n);
 
             // Assert
-            Assert.That(zipped, Has.Count.EqualTo(2));
-            Assert.That(zipped[0], Has.Property("Letter").EqualTo("a").And.Property("Number").EqualTo(1));
-            Assert.That(zipped[1], Has.Property("Letter").EqualTo("b").And.Property("Number").EqualTo(2));
+            zipped.ShouldBeEquivalentTo(new[] { "a1", "b2" });
         }

```

Comment:
The reference redirect thingy made `.ToList()` inaccessible. Or hard to access. When I eyeballed it, I thought, "that's more complicated than it maybe needs to be". So…